clock_exercise/TimerUpDownNumberField.qml

```diff
@@ -0,0 +1,69 @@
+import QtQuick 2.0
+
+Item {
+    id: root
+
+    property alias text: textField.text
+    property int value: 0
+    property int max: 99
+
+    y: 160
+    width: 190
+    height: 502
+
+    Rectangle {
+        width: 190
+        height: 280
+        color: "#1B2F46"
+        radius: 30
+
+        Text {
+            x: 0
+            y: 43
+            width: 190
+            height: 116
+            text: root.value < 10? "0" + root.value.toString(): root.value.toString()
+            color: "#FCB647"
+            font.pixelSize: 120
+            horizontalAlignment: Text.AlignHCenter
+            font.family: "Buenos Aires"
+            font.styleName: "Thin"
+        }
+
+        Text {
+            id: textField
+
+            x: 73
+            y: 215
+            width: 44
+            height: 24
+            color: "#9FAAB0"
+            font.pixelSize: 20
+            horizontalAlignment: Text.AlignHCenter
+            font.family: "Buenos Aires"
+            font.styleName: "Regular"
+        }
+    }
+
+    ArrowButton {
+        id: upButton
+
+        y: 290
+        onClicked: root.value = Math.min(root.value + 1, root.max)
```

Comment:
Qui c'è un problema di range valori, se da fuori usi max a 60 per i minuti, qui ti ritrovi un range che contiene sia 0 che 60, per un totale di 61 possibili minuti diversi.
O prevediamo di usare sempre un max inferiore da fuori (questo lo valuterei in base alla naturalezza e semplicità di utilizzo come utilizzatori) oppure usiamo `root.max - 1` come estremo di riferimento all'interno.

Reply:
Giusto, sistemato, adesso i valori vanno da 0 a 59 per i minuti e da 0 a 99 per le ore